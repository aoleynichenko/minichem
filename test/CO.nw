# Hartree-Fock calculation of CO

start CO
#memory 20 kb
echo
#nproc 8

geometry units atomic
  C 0 0 0
  O 2.16464 0 0
end

basis "ao basis" SPHERICAL
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    SP
      2.9412494             -0.09996723             0.15591627       
      0.6834831              0.39951283             0.60768372       
      0.2222899              0.70011547             0.39195739       
O    S
    130.7093200              0.15432897       
     23.8088610              0.53532814       
      6.4436083              0.44463454       
O    SP
      5.0331513             -0.09996723             0.15591627       
      1.1695961              0.39951283             0.60768372       
      0.3803890              0.70011547             0.39195739       
end

scf
  diis 6
  direct
  maxiter 100
end

task scf
