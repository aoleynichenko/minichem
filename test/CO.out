Minichem 0.0
By Alexander Oleynichenko, 2016
Build:      1281
Build date: Mar  9 2016 21:44:01
Compiler:   gcc 5.2.0
Date:       Wed Mar  9 22:49:03 2016
Cores:      1
Arguments:  ../bin/minichem CO.nw 
------------------------------------------------------------
>>> CO.nw
# Hartree-Fock calculation of CO

start CO
#memory 20 kb
echo
#nproc 4

geometry units atomic
  C 0 0 0
  O 2.16464 0 0
end

basis "ao basis" SPHERICAL
C    S
     71.6168370              0.15432897       
     13.0450960              0.53532814       
      3.5305122              0.44463454       
C    SP
      2.9412494             -0.09996723             0.15591627       
      0.6834831              0.39951283             0.60768372       
      0.2222899              0.70011547             0.39195739       
O    S
    130.7093200              0.15432897       
     23.8088610              0.53532814       
      6.4436083              0.44463454       
O    SP
      5.0331513             -0.09996723             0.15591627       
      1.1695961              0.39951283             0.60768372       
      0.3803890              0.70011547             0.39195739       
end

scf
  diis 7
  maxiter 100
end

task scf
------------------------------------------------------------
                    Basis information
   C
  ----
  1   1S    71.61683700     0.15432897
            13.04509600     0.53532814
             3.53051220     0.44463454

  2   2S     2.94124940    -0.09996723
             0.68348310     0.39951283
             0.22228990     0.70011547

  3   2P     2.94124940     0.15591627
             0.68348310     0.60768372
             0.22228990     0.39195739


   O
  ----
  1   1S   130.70932000     0.15432897
            23.80886100     0.53532814
             6.44360830     0.44463454

  2   2S     5.03315130    -0.09996723
             1.16959610     0.39951283
             0.38038900     0.70011547

  3   2P     5.03315130     0.15591627
             1.16959610     0.60768372
             0.38038900     0.39195739


Spherical basis set summary:
*---------*-----------------*--------*------------*
| Element |   Description   | Shells | Primitives |
*---------*-----------------*--------*------------*
|   C     | ao basis        |   3    |     9      |
|   O     | ao basis        |   3    |     9      |
*---------*-----------------*--------*------------*

------------------------------------------------------------
          ********************************
          *      Parallel SCF Module     *
          ********************************

Formula:   
Atoms:     2
Electrons: 14
Mol mass:  28.010
Charge:    0
Spin mult: 1
Units:     atomic
------------------------------------------------------------
One-electron integrals evaluation algorithm: Obara-Saika
Two-electron integrals evaluation algorithm: Obara-Saika
One-electron integrals done in 0.000001 sec

****************** BASIS ORTHOGONALIZATION *******************
Algorithm: canonical
Basis functions elimination threshold: 1e-08
Overlap matrix lowest eigenvalue = 0.21393818
AO basis orthogonalization done in 0.000067 sec

Initial guess: extended Huckel theory
Initial energy = -134.03920471
Initial guess done in 0.005368 sec

#bfns = 10
#eris = 1540

 iter.       Energy         Delta E       RMS-Dens       DIIS-Err     time
----------------------------------------------------------------------------
   1    -107.45863432     4.40598247     1.97024153     0.68506300    0.06
   2    -110.27895668    -2.82032236     0.78804385     1.11754207    0.12
   3    -111.33923682    -1.06028014     0.20505367     0.22891753    0.19
   4    -111.22278957     0.11644725     0.09275791     0.03542618    0.25
   5    -111.22127881     0.00151076     0.06018666     0.00904010    0.32
   6    -111.22887826    -0.00759945     0.01025002     0.00106578    0.38
   7    -111.22510727     0.00377099     0.00600902     0.00115365    0.44
   8    -111.22541488    -0.00030761     0.00036852     0.00006421    0.51
   9    -111.22545083    -0.00003596     0.00007988     0.00004312    0.57
  10    -111.22544951     0.00000132     0.00000145     0.00000056    0.63
  11    -111.22544951     0.00000000     0.00000001     0.00000001    0.69
----------------------------------------------------------------------------
          Total SCF energy =  -111.22544951
  Nuclear repulsion energy =    22.17458792


      Time for:           sec
---------------------------------
  Orthogonalization      0.000
  Initial guess          0.005
  Density matrix         0.038
  Fock matrix            0.624
  Diagonalization        0.000
---------------------------------


         Molecular Orbitals Summary
  +-----+-----+----------------+----------+
  | No  | Occ |     Energy     | Symmetry |
  +-----+-----+----------------+----------+
  | 1   |  2  |   -20.41556855 |     ?    |
  | 2   |  2  |   -11.09218760 |     ?    |
  | 3   |  2  |    -1.44527065 |     ?    |
  | 4   |  2  |    -0.69683282 |     ?    |
  | 5   |  2  |    -0.53992645 |     ?    |
  | 6   |  2  |    -0.53992645 |     ?    |
  | 7   |  2  |    -0.44513196 |     ?    |
  | 8   |  0  |     0.30614511 |     ?    |
  | 9   |  0  |     0.30614511 |     ?    |
  | 10  |  0  |     1.00910937 |     ?    |
  +-----+-----+----------------+----------+

        Mulliken population analysis
        ----------------------------

      Atom     Population     Charge
      1  C     5.80106445   0.19893555
      2  O     8.19893555  -0.19893555


        Loewdin population analysis
        ---------------------------

      Atom     Population     Charge
      1  C     5.95323785   0.04676215
      2  O     8.04676215  -0.04676215

               Multipole moments
               -----------------
  D       x            y            z
     0.04893493  -0.00000000  -0.00000000

  |D| = 0.04893493 a.u. = 0.12438017 Debye


Allocated: 97996 bytes
Freed:     86036 bytes
Max usage: 21960 bytes
Total time: 0.70 sec
